### ISSUE TITLE
configuration options "help" text seems incorrect

### ISSUE THREAD

**Issue by nmschulte** [Wirena/clang-format-configurator-v2#61]
it seems there are some special leading and trailing codes that are getting included in the help for a particular option; see the
```
:ref ...
```
and
```
.. START_FORMAT_STYLE_OPTIONS
.. _AccessModifierOffset:
```
lines:

![image](https://github.com/user-attachments/assets/9c47009c-2a1d-4610-8bb5-962ddd2d28bf)

**Comment by Wirena** [comment]
@nmschulte yeah, these are leftovers from LLVM rst format doc files, I never bothered to  convert them to HTML properly, so it is what it is for now

Anyway, thanks for letting me know
